tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2823,7 +2823,7 @@ declare namespace ts {
         VariableDeclarationList = 272,
         SingleLineFunctionBodyStatements = 384,
         MultiLineFunctionBodyStatements = 1,
-        ClassHeritageClauses = 256,
+        ClassHeritageClauses = 0,
```

Comment:
Well...this is just `0` now, so it doesn't seem necessary :D

Reply:
Nah. It's only zero because `SingleLine` desugars to zero because reasons (because it's the default). It still carries semantic meaning should we choose to adjust the list format again in the future, as it is a very unique list kind.